pkg/cloud/aws/commands/run_test.go

```diff
@@ -958,8 +971,9 @@ const expectedS3AndCloudTrailResult = `{
 `
 
 func Test_Run(t *testing.T) {
-
-	regoDir := t.TempDir()
+	regoDir := filepath.Join("testdata", "Test_Run_Dir")
+	require.NoError(t, os.MkdirAll(regoDir, 0755))
+	defer os.RemoveAll(regoDir)
```

Comment:
I'm curious about the benefit of this change.

Reply:
https://github.com/open-policy-agent/opa/issues/4521